include/zephyr/usb/msos_desc.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief MS OS 2.0 descriptor definitions
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_USB_MSOS_DESC_H
+#define ZEPHYR_INCLUDE_USB_MSOS_DESC_H
+
+#include <stdint.h>
+
+enum msos2_descriptor_index {
+	MS_OS_20_DESCRIPTOR_INDEX		= 0x07,
+	MS_OS_20_SET_ALT_ENUMERATION		= 0x08,
+};
+
+enum ms_os_20_descriptor_type {
+	MS_OS_20_SET_HEADER_DESCRIPTOR		= 0x00,
+	MS_OS_20_SUBSET_HEADER_CONFIGURATION	= 0x01,
+	MS_OS_20_SUBSET_HEADER_FUNCTION		= 0x02,
+	MS_OS_20_FEATURE_COMPATIBLE_ID		= 0x03,
+	MS_OS_20_FEATURE_REG_PROPERTY		= 0x04,
+	MS_OS_20_FEATURE_MIN_RESUME_TIME	= 0x05,
+	MS_OS_20_FEATURE_MODEL_ID		= 0x06,
+	MS_OS_20_FEATURE_CCGP_DEVICE		= 0x07,
+	MS_OS_20_FEATURE_VENDOR_REVISION	= 0x08
+};
+
+enum ms_os_20_property_data_type {
```

Comment:
Probably my fault, please make it consistent

```c
enum msosv2_descriptor_index {}

enum msosv2_descriptor_type{}

enum msosv2_property_data_type {}
```

Reply:
right, I overlooked those. should be fixed now.